dev/devicelab/bin/tasks/flutter_gallery_macos__start_up.dart

```diff
@@ -2,11 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter_devicelab/framework/devices.dart';
 import 'package:flutter_devicelab/framework/framework.dart';
 import 'package:flutter_devicelab/tasks/perf_tests.dart';
 
 Future<void> main() async {
-  // It's intended to use the Gallery startup test as a smoke test on macOS
-  // Catalina.
```

Comment:
`dev/devicelab/bin/tasks/flutter_gallery_mac__start_up.dart` test was for Android `Mac_android flutter_gallery_mac__start_up`.  The rename to `flutter_gallery_macos__start_up.dart` broke that test.

https://github.com/flutter/flutter/blob/4770e348595551832a4762ea0eabe2cca59b668c/.ci.yaml#L3084